src/main/kotlin/com/kafkakotlin/demo/users/UserService.kt

```diff
@@ -0,0 +1,21 @@
+package com.kafkakotlin.demo.users
+
+import org.springframework.stereotype.Service
+
+@Service
+class UserService(
+    val userRepository: UserRepository
+) {
+
+    fun createUser(user: User) {
```

Comment:
For simple methods like this you can get rid of the braces + the return and just do
`fun createUser(user: User) = userRepository.createUser(user)`